src/dotnet/APIView/ClientSPA/src/app/_components/revisions-list/revisions-list.component.ts

```diff
@@ -544,7 +544,7 @@ export class RevisionsListComponent implements OnInit, OnChanges {
           `Use <code>api-extractor</code> to generate a <a href="https://api-extractor.com/pages/setup/generating_docs/">docModel file</a>`,
           `Upload generated api.json file`
         ];
-        this.acceptedFilesForReviewUpload = ".api.json";
```

Comment:
This shouldn't be changes. JavaScript must upload the .api.json file which is the output of api-extractor tool they use.

Reply:
`.api.json` is not actually an extension so it has to be `.json` or else the upload will not work. The instruction still says upload `.api.json`